packages/alfa-style/test/property/mask-clip.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import { test } from "@siteimprove/alfa-test";
+import { h } from "@siteimprove/alfa-dom";
+
+import { computed } from "../common.js";
+
+test("initial value is border-box", (t) => {
+  const element = <div></div>;
+
+  t.deepEqual(computed(element, "mask-clip"), {
+    value: {
+      type: "list",
+      separator: " ",
+      values: [
+        {
+          type: "keyword",
+          value: "border-box",
+        },
+      ],
```

Comment:
```suggestion
      values: [ { type: "keyword", value: "border-box" } ],
```
When prettier is happy with it, I tend to prefer avoiding delimiters-only lines in tests 🙂 (but it's a bit annoying to do the switch everywhere, so feel free to ignore it…)

(alternatively / additionally, it might be worth testing `computed("foo").value.values` just to get rid of the list wrapper (it needs to be kept once to check the separator) and reduce the indentation level and number of empty-ish lines)